plugins/commands/reporter/src/main/kotlin/ReporterCommand.kt

```diff
@@ -262,16 +263,16 @@ class ReporterCommand : OrtCommand(
             howToFixTextProvider
         )
 
-        val reportOptionsMap = sortedMapOf<String, MutableMap<String, String>>(String.CASE_INSENSITIVE_ORDER)
+        val reportOptionsMap = sortedMapOf<String, PluginConfiguration>(String.CASE_INSENSITIVE_ORDER)
 
-        ortConfig.reporter.options?.forEach { (reporterName, option) ->
-            val reportSpecificOptionsMap = reportOptionsMap.getOrPut(reporterName) { mutableMapOf() }
-            reportSpecificOptionsMap += option
+        ortConfig.reporter.config?.forEach { (reporterName, config) ->
+            reportOptionsMap[reporterName] = config
         }
 
         reportOptions.forEach { (reporterName, option) ->
-            val reportSpecificOptionsMap = reportOptionsMap.getOrPut(reporterName) { mutableMapOf() }
-            reportSpecificOptionsMap[option.first] = option.second
+            val reportSpecificConfig = reportOptionsMap.getOrPut(reporterName) { PluginConfiguration.EMPTY }
+            val updatedConfig = reportSpecificConfig.copy(options = reportSpecificConfig.options + option)
```

Comment:
Maybe add an explaining code comment, saying that this makes CLI options overwrite options from the configuration?